app/services/encrypted_document_storage/document_writer.rb

```diff
@@ -34,7 +34,7 @@ def storage
     end
 
     def aes_cipher
-      @aes_cipher ||= Encryption::AesCipher.new
+      @aes_cipher ||= Encryption::LegacyAesCipher.new
```

Comment:
I would consider calling these `AesCipherV2` or something (same ideas as planning for an eventual v4)